src/HLL/Compiler.nqp

```diff
@@ -55,7 +52,7 @@ class HLL::Compiler does HLL::Backend::Default {
         nqp::getcomp($name);
     }
 
-    method config() { %!config };
+    method config() { nqp::gethllsym('default', 'SysConfig').nqp-build-config }
```

Comment:
A very minor note: the method name confused me into thinking that the config would be rebuilt upon every `.config()` call.

Reply:
I doubt if we actually need the `config()` method with `nqp::gethllsym('default', 'SysConfig')` available. It's current name suggest that it's compiler's configuration.

I would consider something like `sysconfig`, or `sys-config`, or `system-config`. Perhaps the first one is the best due to its consonance with with the hllsym and class name.

Reply:
Done.